zoomChatDecoderVer2.py

```diff
@@ -0,0 +1,95 @@
+
+print("Decoding Zoom chat .txt-s")
+print("Enter the name (or path) of your .txt file!")
+print("Examples: myFile.txt OR C:/Documents/meeting_saved_chat.txt")

```

Comment:
Print an info that it's enough just to click Enter if the file has got a default name (meeting_saved_chat.txt) and it's in the same directory as the Python code. Python file is very light so the user can make a lot of copies and paste them into the desired directories, then just type: Enter -> "save" (-> "end").